ml-testing-toolkit-cli/templates/job.yaml

```diff
@@ -26,7 +26,7 @@
               wget {{ .Values.config.testCasesZipUrl }} -O downloaded-test-collections.zip;
               mkdir tmp_test_cases;
               unzip -d tmp_test_cases -o downloaded-test-collections.zip;
-              npm run cli -- -c cli-default-config.json -e cli-testcase-environment.json -i tmp_test_cases/{{ .Values.config.testCasesPathInZip }} -u {{ .Values.config.ttkBackendURL }} --report-format html --report-target s3://{{ .Values.config.awsS3BucketName }}/{{ .Values.config.awsS3FilePath }}/asdf.html --report-auto-filename-enable true --slack-webhook-url $SLACK_WEBHOOK_URL --extra-summary-information="Test Suite:{{ .Values.config.testSuiteName }},Environment:{{ .Values.config.environmentName }}";
+              npm run cli -- -c cli-default-config.json -e cli-testcase-environment.json -i tmp_test_cases/{{ .Values.config.testCasesPathInZip }} -u {{ .Values.config.ttkBackendURL | replace "$release_name" .Release.Name }} --report-format html --report-target s3://{{ .Values.config.awsS3BucketName }}/{{ .Values.config.awsS3FilePath }}/asdf.html --report-auto-filename-enable true --slack-webhook-url $SLACK_WEBHOOK_URL --extra-summary-information="Test Suite:{{ .Values.config.testSuiteName }},Environment:{{ .Values.config.environmentName }}";
```

Comment:
Just to confirm. What happens if the following configuration values are not defined or null?

- .Values.config.awsS3BucketName
- $SLACK_WEBHOOK_URL

Reply:
Slack notification and s3 upload will be failed if we don't supply the values externally. Because it uses the default values. Shall I provide "if condition" there to not to pass the arguments if the values are null?

Reply:
That would be great!

Reply:
By default, they should be null/undefined.